packages/expo-splash-screen/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### 🛠 Breaking changes
 
 ### 🎉 New features
+- Add warning for splash screen if visible for too long ([#12882](https://github.com/expo/expo/pull/12882) by [@ajsmth](https://github.com/ajsmth))
 
```

Comment:
another little spacing nit - newline should be after header and no extra newline between new feature items in the changelog list